.github/workflows/functional-tests-approval.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Approve Functional Tests'
+on:
+  pull_request:
+    branches:
+      - main
+      - features/*
+      - release/*
+jobs:
+  wait_for_approval:
+    runs-on: ubuntu-latest
+    environment: functional-tests
+    steps:
```

Comment:
Added an extra step because you cannot use environment with reusable workflow

Reply:
Can you explain more? I'm not sure what you mean. I thought you were writing composite actions?